DOC Update Getting Started and ORM sections (#189)
silverstripe/developer-docs: en/00_Getting_Started/03_Environment_Management.md

need to do so manually. See the [Including an extra `.env` file](#including-an-extra-env-file) section below for more
information.

## Other ways to manage environment variables

> Silverstripe CMS will respect "real" environment variables provided by the server.

Reviewer: ```suggestion
Silverstripe CMS will respect environment variables provided by the server.
```

Reviewer: Also remove the mention to "real" environment variables in the apache section below

Reviewer 2: That wording was pre-existing but I have no problem removing it. Done.

Environment::getEnv('SS_DATABASE_CLASS');
```

[hint]

> The `Environment::getEnv()` method will return `false` both when there was no value set for a variable and when

Reviewer: ```suggestion
The `Environment::getEnv()` method will return `false` both if there was no value set for a variable or if
```
Just make it a little clear they're mutually exclusive reasons

Reviewer 2: Done

$loader->loadFile($env);
```

[warning]

> Note that because `_config.php` is processed after yaml configuration, variables set in these extra `.env` files cannot be used inside yaml config.

Reviewer: They can?

e.g. https://github.com/silverstripe/silverstripe-framework/blob/5.0/_config/mailer.yml#L28

Reviewer 2: Note this is _explicitly referring to extra .env files_ - not the normal `.env` file that we use by default.

| `SS_MANIFESTCACHE` | The manifest cache to use (defaults to file based caching). Must be a `Psr\Cache\CacheItemPoolInterface`, `Psr\SimpleCache\CacheInterface`, or `SilverStripe\Core\Cache\CacheFactory` class implementation. |
| `SS_IGNORE_DOT_ENV` | If set, the `.env` file will be ignored. This is good for live to mitigate any performance implications of loading the `.env` file. |
| `SS_BASE_URL` | The url to use when it isn't determinable by other means (eg: for CLI commands). Should either start with a protocol (e.g. `https://www.example.com`) or with a double forward slash (e.g. `//www.example.com`). |
| `SS_FLUSH_ON_DEPLOY` | Try to detect deployments through file system modifications and flush on the first request after every deploy. Does not run "dev/build" - only "flush". Possible values are `true` (check for a framework PHP file modification time), `false` (no checks, skip deploy detection) or a path to a specific file or folder to be checked. See [DeployFlushDiscoverer](api:SilverStripe\Core\Startup\DeployFlushDiscoverer) for more details.<br /><br />False by default. |

> | `SS_TEMP_PATH` | File storage used for the default cache adapters in [Manifests](/developer_guides/execution_pipeline/manifests), [Object Caching](/developer_guides/performance/caching) and [Partial Template Caching](/developer_guides/templates/partial_template_caching). Can be an absolute path (with a leading `/`), or a path relative to the project root. Defaults to creating a sub-directory of PHP's built-in `sys_get_temp_dir()` or using the `silverstripe-cache` directory relative to the project root if one is present. |

Reviewer: Just wondering if it's worth alpha sorting all of these

Reviewer 2: That's out of scope for this PR.
